### PR TITLE
feat(terminal): added ZSH shell for Linux based OS

### PR DESCRIPTION
**Pull request by dipankardas011**
Signed-off-by: Dipankar Das <dipankardas0115@gmail.com>

This PR...

## Changes

- Addition of ZSH shell for the 'other' OS

## Fixes

- Issue #2306 

## Screenshots

Screenshot from 2022-08-30 22-47-02

## Checklist

- [X] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/constants/terminal.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const SUPPORTED_SHELLS = {
```
Comment (topliceanurazvan) [inline src/constants/terminal.ts:1]: Let's make the SUPPORTED_SHELLS a record of `<string, string[]>`

File: src/redux/services/terminalShells.ts [hunk 2]
```diff
@@ -28,6 +28,7 @@ const detectAvailableUnixShells = (osPlatform: NodeJS.Platform): ShellsMapType =
     });
   } else {
```
Comment (topliceanurazvan) [inline src/redux/services/terminalShells.ts:29]: Here we should check the same as for the `darwin` platform and only add the `zsh` and `bash` if they are found in `/etc/shells` folder

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]
Hello @dipankardas011 and thank you for contributing to Monokle! 🙌🏻 

I've left some comments regarding your PR, if you have any questions please let us know.

File: src/constants/terminal.ts (hunk 1 above)
Comment (dipankardas011) [reply src/constants/terminal.ts:1]: ig it's is in this form
image

File: src/redux/services/terminalShells.ts (hunk 2 above)
Comment (dipankardas011) [reply src/redux/services/terminalShells.ts:29]: Got it going 
image

File: src/constants/terminal.ts (hunk 1 above)
Comment (dipankardas011) [reply src/constants/terminal.ts:1]: ig its already resolved just added some more shells which many people use

**Comment by dipankardas011** [comment]
@topliceanurazvan changed as you suggested

**Review by topliceanurazvan — APPROVED** [review]
Great work! 🙌🏻

**Comment by dipankardas011** [comment]
Thanks for your review

**Comment by chargio** [comment]
Great!

**Pull request merged**